metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/SensorEnrichmentConfigController.java

```diff
@@ -50,6 +56,7 @@
   @RequestMapping(value = "/{name}", method = RequestMethod.POST)
   ResponseEntity<SensorEnrichmentConfig> save(@ApiParam(name="name", value="SensorEnrichmentConfig name", required=true)@PathVariable String name,
                                               @ApiParam(name="sensorEnrichmentConfig", value="SensorEnrichmentConfig", required=true)@RequestBody SensorEnrichmentConfig sensorEnrichmentConfig) throws RestException {
+    LOG.info(String.format("User '%s' changed the '%s' enrichment config to %s", SecurityUtils.getCurrentUser(), name, sensorEnrichmentConfig.toString()));
```

Comment:
We should be using slf4j tokenization instead of String.format()

```
LOG.info("User '{}' changed the '{}' enrichment config to {}", SecurityUtils.getCurrentUser(), name, sensorEnrichmentConfig.toString());
```